gpkit/tests/t_model.py

```diff
@@ -386,7 +387,8 @@ def test_small_named_signomial(self):
         nonzero_adder = 0.1  # TODO: support reaching zero, issue #348
         with SignomialsEnabled():
             J = 0.01*(x - 1)**2 + nonzero_adder
-            m = Model(z, [z >= J], name="SmallSignomial")
+            with NamedVariables("SmallSignomial"):
+                m = Model(z, [z >= J])
```

Comment:
Yeah, the thought was that users might want it? But I think you're right, I should remove it.